encrypted-media-respec.html

```diff
@@ -3611,7 +3612,7 @@
 
             <dt>User deletion of Key System storage</dt>
             <dd>
-              <p>User agents SHOULD present the user with a way to delete Key System storage for a specific <a def-id="origin"></a> or all origins.
+              <p>User agents SHOULD present the user with a way to specifically delete Key System storage, by <a def-id="origin"></a> or for all origins.
```

Comment:
"or" seems correct.  The way is provided by origin OR for all origins.  "by origin" seems encompassed by "all origins".